backend/census_historical_migration/loader.py

```diff
@@ -0,0 +1,28 @@
+from audit.models import SingleAuditChecklist
+from .models import ELECAUDITHEADER as Gen
+from .make_submission import load_historic_data
+
+
+def load_data(audit_yaer):
+    SingleAuditChecklist.objects.all().delete()
```

Comment:
I know this is still in draft, but this absolutely cannot go to production as-is. This line is terrifying.